MAVSDK-CSharp/MAVSDK.CSharp.ConsoleClient/Program.cs

```diff
@@ -2,7 +2,7 @@
 using System.Reactive;
 using System.Reactive.Linq;
 using System.Threading.Tasks;
-using MAVSDK_CSharp.Plugins;
+using Action = MAVSDK_CSharp.Plugins.Action;
```

Comment:
I know this is bad, in the future I will edit `protoc-gen-dcsdk` so that we can add a suffix to the plugins, and they will be called `ActionPlugin`, `CameraPlugin`, etc.

Reply:
Sure thing!